Can't create file when saving to CSV when title has `/` in it

**Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!**

**Describe the bug**
When saving a response with "Save CSV to file" there's no option to create a new file in Nextcloud.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'Results'
2. Click on 'Save SCV to file'
3. Choose a folder. 
4. You have to select a file that's already there. (Clicking the + button creates a new folder. )

**Expected behavior**
Expected behavior would be a button that says "Create new file", and then the option to give the new file a name.

**Screenshots**
Screenshot shows the Save to CSV-file window.
![Screenshot 2023-05-23 at 13-32-59 Rapport fra foredrag_innledning_debatt - Skjemaer - Nextcloud - Attac](https://github.com/nextcloud/forms/assets/32357781/798518fd-f3a7-4dd6-a344-caddc897a1d1)


**Nextcloud (please complete the following information):**
- Nextcloud-Version: 26.0.1
- Forms-Version: 3.3.0

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser : Firefox
 - Version: 113.0.1

The error warnings says I get: "Det oppstod en feil ved eksportering til filer" (the setup is in Norwegian) which translates to "There was an error when exporting to files". 

Nextcloud logs (it says _Feil_ which is Norwegian for Error):
```
[index] Feil: TypeError: OCA\Files\Service\TagService::__construct(): Argument #4 ($homeFolder) must be of type OCP\Files\Folder, null given, called in /var/www/nextcloud/apps/files/lib/AppInfo/Application.php on line 111 at <<closure>>

 0. /var/www/nextcloud/apps/files/lib/AppInfo/Application.php line 111
    OCA\Files\Service\TagService->__construct()
 1. /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php line 171
    OCA\Files\AppInfo\Application->OCA\Files\AppInfo\{closure}("*** sensitive parameters replaced ***")
 2. /var/www/nextcloud/3rdparty/pimple/pimple/src/Pimple/Container.php line 122
    OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}("*** sensitive parameters replaced ***")
 3. /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php line 138
    Pimple\Container->offsetGet()
 4. /var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php line 487
    OC\AppFramework\Utility\SimpleContainer->query()
 5. /var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php line 465
    OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback()
 6. /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php line 65
    OC\AppFramework\DependencyInjection\DIContainer->query()
 7. /var/www/nextcloud/apps/files/lib/AppInfo/Application.php line 90
    OC\AppFramework\Utility\SimpleContainer->get()
 8. /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php line 171
    OCA\Files\AppInfo\Application->OCA\Files\AppInfo\{closure}("*** sensitive parameters replaced ***")
 9. /var/www/nextcloud/3rdparty/pimple/pimple/src/Pimple/Container.php line 122
    OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}("*** sensitive parameters replaced ***")
10. /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php line 138
    Pimple\Container->offsetGet()
11. /var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php line 487
    OC\AppFramework\Utility\SimpleContainer->query()
12. /var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php line 465
    OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback()
13. /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php line 65
    OC\AppFramework\DependencyInjection\DIContainer->query()
14. /var/www/nextcloud/lib/private/AppFramework/App.php line 148
    OC\AppFramework\Utility\SimpleContainer->get()
15. /var/www/nextcloud/lib/private/Route/Router.php line 315
    OC\AppFramework\App::main()
16. /var/www/nextcloud/lib/base.php line 1056
    OC\Route\Router->match()
17. /var/www/nextcloud/index.php line 36
    OC::handleRequest()

GET /apps/files/api/v1/stats
from 77.40.167.80 at 2023-05-24T07:25:52+00:00
``` 


```
[richdocuments] Feil: OCA\Richdocuments\Exceptions\UnknownTokenException: Could not find token. at <<closure>>

0. /var/www/nextcloud/apps/richdocuments/lib/Middleware/WOPIMiddleware.php line 76
   OCA\Richdocuments\Db\WopiMapper->getWopiForToken()
1. /var/www/nextcloud/lib/private/AppFramework/Middleware/MiddlewareDispatcher.php line 96
   OCA\Richdocuments\Middleware\WOPIMiddleware->beforeController()
2. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 129
   OC\AppFramework\Middleware\MiddlewareDispatcher->beforeController()
3. /var/www/nextcloud/lib/private/AppFramework/App.php line 183
   OC\AppFramework\Http\Dispatcher->dispatch()
4. /var/www/nextcloud/lib/private/Route/Router.php line 315
   OC\AppFramework\App::main()
5. /var/www/nextcloud/lib/base.php line 1056
   OC\Route\Router->match()
6. /var/www/nextcloud/index.php line 36
   OC::handleRequest()

GET /index.php/apps/richdocuments/wopi/files/3564_ochcrvrelgqi?access_token=2jgbzApMy2pkSsmzUFjwGRHkyCEVhWcf&access_token_ttl=1684890164000&permission=edit
from 185.220.119.250 at 2023-05-24T07:29:38+00:00
```
```
[richdocuments] Feil: OCA\Richdocuments\Exceptions\UnknownTokenException: Could not find token. at <<closure>>

0. /var/www/nextcloud/apps/richdocuments/lib/Middleware/WOPIMiddleware.php line 76
   OCA\Richdocuments\Db\WopiMapper->getWopiForToken()
1. /var/www/nextcloud/lib/private/AppFramework/Middleware/MiddlewareDispatcher.php line 96
   OCA\Richdocuments\Middleware\WOPIMiddleware->beforeController()
2. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 129
   OC\AppFramework\Middleware\MiddlewareDispatcher->beforeController()
3. /var/www/nextcloud/lib/private/AppFramework/App.php line 183
   OC\AppFramework\Http\Dispatcher->dispatch()
4. /var/www/nextcloud/lib/private/Route/Router.php line 315
   OC\AppFramework\App::main()
5. /var/www/nextcloud/lib/base.php line 1056
   OC\Route\Router->match()
6. /var/www/nextcloud/index.php line 36
   OC::handleRequest()

GET /index.php/apps/richdocuments/wopi/files/3733_ochcrvrelgqi?access_token=SYlcoQWR8y22FVoSUNvFxFWkrlrFwCgQ&access_token_ttl=1684891323000&permission=edit
from 185.220.119.250 at 2023-05-24T07:31:57+00:00
```

```
[richdocuments] Feil: OCA\Richdocuments\Exceptions\UnknownTokenException: Could not find token. at <<closure>>

0. /var/www/nextcloud/apps/richdocuments/lib/Middleware/WOPIMiddleware.php line 76
   OCA\Richdocuments\Db\WopiMapper->getWopiForToken()
1. /var/www/nextcloud/lib/private/AppFramework/Middleware/MiddlewareDispatcher.php line 96
   OCA\Richdocuments\Middleware\WOPIMiddleware->beforeController()
2. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 129
   OC\AppFramework\Middleware\MiddlewareDispatcher->beforeController()
3. /var/www/nextcloud/lib/private/AppFramework/App.php line 183
   OC\AppFramework\Http\Dispatcher->dispatch()
4. /var/www/nextcloud/lib/private/Route/Router.php line 315
   OC\AppFramework\App::main()
5. /var/www/nextcloud/lib/base.php line 1056
   OC\Route\Router->match()
6. /var/www/nextcloud/index.php line 36
   OC::handleRequest()

GET /index.php/apps/richdocuments/wopi/files/3564_ochcrvrelgqi?access_token=2jgbzApMy2pkSsmzUFjwGRHkyCEVhWcf&access_token_ttl=1684890164000&permission=edit
from 185.220.119.250 at 2023-05-24T07:32:01+00:00
```

Browser logs:
```
Browser Console Mode
Parent process only(Fast)
Multiprocess(Slower)
DOMException: Could not parse path (null): NS_ERROR_FILE_UNRECOGNIZED_PATH
    _getChromeUserDataPathIfExists resource:///modules/ChromeProfileMigrator.sys.mjs:109
[MigratorBase.sys.mjs:562:15](resource:///modules/MigratorBase.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152206	app.normandy.recipe-runner	ERROR	Error checking filter for "Heartbeat: Daily Viewpoint Survey (pt-BR)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["pt-BR"]) && (['rolling-viewpoint', normandy.userId]|bucketSample(19468 + normandy.request_time / (24*60*60*1000) + 7, 7, 7000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152220	app.normandy.recipe-runner	ERROR	Error checking filter for "Heartbeat: Daily Viewpoint Survey (ja)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["ja","ja-JP-mac"]) && (['rolling-viewpoint', normandy.userId]|bucketSample(19468 + normandy.request_time / (24*60*60*1000) + 7, 7, 7000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152233	app.normandy.recipe-runner	ERROR	Error checking filter for "Heartbeat: Daily Viewpoint Survey (zh-CN)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["zh-CN"]) && (['rolling-viewpoint', normandy.userId]|bucketSample(19468 + normandy.request_time / (24*60*60*1000) + 7, 7, 7000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152247	app.normandy.recipe-runner	ERROR	Error checking filter for "Heartbeat: Daily Viewpoint Survey (zh-TW)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["zh-TW"]) && (['rolling-viewpoint', normandy.userId]|bucketSample(19468 + normandy.request_time / (24*60*60*1000) + 7, 7, 7000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152266	app.normandy.recipe-runner	ERROR	Error checking filter for "Heartbeat: Daily Viewpoint Survey (en)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["en-AU","en-GB","en-CA","en-NZ","en-ZA","en-US"]) && (['rolling-viewpoint', normandy.userId]|bucketSample(19468 + normandy.request_time / (24*60*60*1000) + 7, 7, 7000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.country [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152338	app.normandy.recipe-runner	ERROR	Error checking filter for "HB: New Profiles Phase 2 (Germany / de / gen pop)". Filter: [(normandy.channel in ["release"]) && (normandy.country in ["DE"]) && (normandy.locale in ["de"]) && (["global-v5",normandy.userId]|bucketSample(6800,300,10000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.country [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152354	app.normandy.recipe-runner	ERROR	Error checking filter for "HB: New Profiles Phase 2 (Germany / de / first use < 1 month ago)". Filter: [(normandy.channel in ["release"]) && (normandy.country in ["DE"]) && (["global-v5",normandy.userId]|bucketSample(7800,500,10000)) && (normandy.locale in ["de"]) && (!normandy.isFirstRun && (normandy.request_time / (1000 * 60 * 60 * 24) - normandy.telemetry.main.environment.profile.firstUseDate < 30))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.country [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152378	app.normandy.recipe-runner	ERROR	Error checking filter for "HB: New Profiles Phase 2 (Germany / de / first use < 3 months ago)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["de","de-AT","de-DE","de-CH"]) && (normandy.country in ["DE"]) && (["global-v5",normandy.userId]|bucketSample(8800,200,10000)) && (!normandy.isFirstRun && (normandy.request_time / (1000 * 60 * 60 * 24) - normandy.telemetry.main.environment.profile.firstUseDate < 90))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.country [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152395	app.normandy.recipe-runner	ERROR	Error checking filter for "HB: New Profiles Phase 2 (France / fr / gen pop)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["fr"]) && (normandy.country in ["FR"]) && (["global-v5",normandy.userId]|bucketSample(6800,300,10000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.country [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152418	app.normandy.recipe-runner	ERROR	Error checking filter for "HB: New Profiles Phase 2 (France / fr / first use < 1 month ago)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["fr"]) && (normandy.country in ["FR"]) && (["global-v5",normandy.userId]|bucketSample(7800,500,10000)) && (!normandy.isFirstRun && (normandy.request_time / (1000 * 60 * 60 * 24) - normandy.telemetry.main.environment.profile.firstUseDate < 30))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.country [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152445	app.normandy.recipe-runner	ERROR	Error checking filter for "HB: New Profiles Phase 2 (France / fr / first use < 3 months ago)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["fr"]) && (normandy.country in ["FR"]) && (["global-v5",normandy.userId]|bucketSample(8800,200,10000)) && (!normandy.isFirstRun && (normandy.request_time / (1000 * 60 * 60 * 24) - normandy.telemetry.main.environment.profile.firstUseDate < 90))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.country [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152461	app.normandy.recipe-runner	ERROR	Error checking filter for "HB: New Profiles Phase 2 (US / en / gen pop)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["en-US"]) && (normandy.country in ["US"]) && (["global-v5",normandy.userId]|bucketSample(6800,200,10000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.country [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152483	app.normandy.recipe-runner	ERROR	Error checking filter for "HB: New Profiles Phase 2 (US / en / first use < 1 month ago)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["en-US"]) && (normandy.country in ["US"]) && (["global-v5",normandy.userId]|bucketSample(7800,500,10000)) && (!normandy.isFirstRun && (normandy.request_time / (1000 * 60 * 60 * 24) - normandy.telemetry.main.environment.profile.firstUseDate < 30))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152520	app.normandy.recipe-runner	ERROR	Error checking filter for "Heartbeat: Daily Viewpoint Survey (fa)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["fa"]) && (['rolling-viewpoint', normandy.userId]|bucketSample(19468 + normandy.request_time / (24*60*60*1000) + 7, 7, 7000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.request_time [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152545	app.normandy.recipe-runner	ERROR	Error checking filter for "Heartbeat: Daily Viewpoint Survey (it)". Filter: [(normandy.channel in ["release"]) && (normandy.locale in ["it"]) && (['rolling-viewpoint', normandy.userId]|bucketSample(19468 + normandy.request_time / (24*60*60*1000) + 7, 7, 7000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.country [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152709	app.normandy.recipe-runner	ERROR	Error checking filter for "pref-rollout-activity-stream-pkt-new-tab-release69-layout-1577008". Filter: [(normandy.locale in ["en-US"]) && (normandy.country in ["US"]) && (normandy.channel in ["release"]) && ((env.version|versionCompare("69.!")>=0)&&(env.version|versionCompare("69.*")<0)) && ([729,normandy.userId]|bucketSample(639,940,1000))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
attribute_error: normandy.country [Targeting.sys.mjs:130:15](resource://messaging-system/targeting/Targeting.sys.mjs)
TypeError: NetworkError when attempting to fetch resource. [Targeting.sys.mjs:153:21](resource://messaging-system/targeting/Targeting.sys.mjs)
1684913152749	app.normandy.recipe-runner	ERROR	Error checking filter for "DoH US Staged Rollout to All US Desktop Users". Filter: [(normandy.country in ["US"]) && (normandy.channel in ["release"]) && (["doh-rollout",normandy.userId]|bucketSample(0,9900,10000)) && ((env.version|versionCompare("76.!")>=0)&&(env.version|versionCompare("91.*")<0))]. Error: "TypeError: NetworkError when attempting to fetch resource." [Log.sys.mjs:722](resource://gre/modules/Log.sys.mjs)
BroadcastService: receivedBroadcastMessage: handler for remote-settings/monitor_changes threw error: Error: Polling for changes failed: NetworkError: Network request failed.
    pollChanges resource://services-settings/remote-settings.sys.mjs:324
[PushBroadcastService.sys.mjs:260](resource://gre/modules/PushBroadcastService.sys.mjs)
services.settings: NetworkError: Network error: TypeError: NetworkError: Network request failed
    APIError resource://services-settings/RemoteSettingsClient.sys.mjs:111
    NetworkError resource://services-settings/RemoteSettingsClient.sys.mjs:115
    _adjustedError resource://services-settings/RemoteSettingsClient.sys.mjs:911
    maybeSync resource://services-settings/RemoteSettingsClient.sys.mjs:816
Caused by: TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.jsm:237
[remote-settings.sys.mjs:391](resource://services-settings/remote-settings.sys.mjs)
BroadcastService: receivedBroadcastMessage: handler for remote-settings/monitor_changes threw error: NetworkError: Network error: TypeError: NetworkError: Network request failed
    APIError resource://services-settings/RemoteSettingsClient.sys.mjs:111
    NetworkError resource://services-settings/RemoteSettingsClient.sys.mjs:115
    _adjustedError resource://services-settings/RemoteSettingsClient.sys.mjs:911
    maybeSync resource://services-settings/RemoteSettingsClient.sys.mjs:816
Caused by: TypeError: NetworkError: Network request failed
    onerror resource://services-settings/Utils.jsm:237
[PushBroadcastService.sys.mjs:260](resource://gre/modules/PushBroadcastService.sys.mjs)
Sync encountered an error - see about:sync-log for the log file. [policies.sys.mjs:977:15](resource://services-sync/policies.sys.mjs)
Element.releaseCapture() er foreldet. Bruk Element.releasePointerCapture() i stedet. For mer hjelp, se https://developer.mozilla.org/docs/Web/API/Element/releasePointerCapture [menupopup.js:160:13](chrome://global/content/elements/menupopup.js)
```

Note, I'm on a hosted plan for Nextcloud.

--- a/lib/Service/SubmissionService.php
+++ b/lib/Service/SubmissionService.php
@@ -261,6 +261,8 @@ class SubmissionService {
 
 		// TRANSLATORS Appendix for CSV-Export: 'Form Title (responses).csv'
 		$fileName = $form->getTitle() . ' (' . $this->l10n->t('responses') . ').csv';
+		// Sanitize file name, replace all invalid characters
+		$fileName = str_replace(mb_str_split(\OCP\Constants::FILENAME_INVALID_CHARS), '-', $fileName);
 
 		return [
 			'fileName' => $fileName,

--- a/tests/Unit/Service/SubmissionServiceTest.php
+++ b/tests/Unit/Service/SubmissionServiceTest.php
@@ -193,25 +193,27 @@ class SubmissionServiceTest extends TestCase {
 
 	public function dataWriteCsvToCloud() {
 		return [
-			'rootFolder' => ['', Folder::class, '', 'Some nice Form Title (responses).csv', false],
-			'subFolder' => ['/folder path', Folder::class, '', 'Some nice Form Title (responses).csv', false],
-			'nonCsv-file' => ['/fileName.txt', File::class, 'txt', 'Some nice Form Title (responses).csv', false],
-			'csv-file' => ['/fileName.csv', File::class, 'csv', 'fileName.csv', true],
+			'rootFolder' => ['Some nice Form Title', '', Folder::class, '', 'Some nice Form Title (responses).csv', false],
+			'subFolder' => ['Some nice Form Title', '/folder path', Folder::class, '', 'Some nice Form Title (responses).csv', false],
+			'nonCsv-file' => ['Some nice Form Title', '/fileName.txt', File::class, 'txt', 'Some nice Form Title (responses).csv', false],
+			'csv-file' => ['Some nice Form Title', '/fileName.csv', File::class, 'csv', 'fileName.csv', true],
+			'invalidFormTitle' => ['Form 1 / 2', '', Folder::class, '', 'Form 1 - 2 (responses).csv', false],
 		];
 	}
 
 	/**
 	 * @dataProvider dataWriteCsvToCloud
 	 *
+	 * @param string $formTitle Given form title
 	 * @param string $path Selected user-path (from frontend)
 	 * @param string $pathClass Type of $path - Folder or File
 	 * @param string $pathExtension Extension of the given file within path
 	 * @param string $expectedFileName
 	 * @param bool $fileExists If the file to write into does exist already.
 	 */
-	public function testWriteCsvToCloud(string $path, string $pathClass, string $pathExtension, string $expectedFileName, bool $fileExists) {
+	public function testWriteCsvToCloud(string $formTitle, string $path, string $pathClass, string $pathExtension, string $expectedFileName, bool $fileExists) {
 		// Simple default Form Data here, details are tested in testGetSubmissionsCsv
-		$dataExpectation = $this->setUpSimpleCsvTest();
+		$dataExpectation = $this->setUpSimpleCsvTest($formTitle);
 
 		$fileNode = $this->createMock(File::class);
 		$fileNode->expects($this->once())
@@ -416,7 +418,7 @@ class SubmissionServiceTest extends TestCase {
 	 * @param string $csvText
 	 */
 	public function testGetSubmissionsCsv(array $questions, array $submissions, string $csvText) {
-		$dataExpectation = $this->setUpCsvTest($questions, $submissions, $csvText);
+		$dataExpectation = $this->setUpCsvTest($questions, $submissions, $csvText, 'Some nice Form Title');
 
 		$this->assertEquals([
 			'fileName' => 'Some nice Form Title (responses).csv',
@@ -427,7 +429,7 @@ class SubmissionServiceTest extends TestCase {
 	/**
 	 * Setting up a very simple default CsvTest
 	 */
-	private function setUpSimpleCsvTest(): string {
+	private function setUpSimpleCsvTest(string $formTitle = 'Some nice Form Title'): string {
 		return $this->setUpCsvTest(
 			[
 				// Single Question
@@ -448,18 +450,20 @@ class SubmissionServiceTest extends TestCase {
 			'
 			"User ID","User display name","Timestamp","Question 1"
 			"user1","User 1","1973-11-29T22:33:09+01:00","Q1A1"
-			'
+			',
+			// Form title
+			$formTitle
 		);
 	}
 
 	/**
 	 * Setting up all the mock-data for a full Form incl. Submissions
 	 */
-	private function setUpCsvTest(array $questions, array $submissions, string $csvText): string {
+	private function setUpCsvTest(array $questions, array $submissions, string $csvText, string $formTitle): string {
 		$form = new Form();
 		$form->setId(5);
 		$form->setHash('abcdefg');
-		$form->setTitle('Some nice Form Title');
+		$form->setTitle($formTitle);
 		$this->formMapper->expects($this->any())
 			->method('findByHash')
 			->with('abcdefg')

